src/transformers/modeling_utils.py

```diff
@@ -1049,6 +1114,10 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         from_pipeline = kwargs.pop("_from_pipeline", None)
         from_auto_class = kwargs.pop("_from_auto", False)
         _fast_init = kwargs.pop("_fast_init", True)
+        torch_dtype = kwargs.pop("torch_dtype", None)
+        torch_dtype_auto_detect = kwargs.pop("torch_dtype_auto_detect", False)
```

Comment:
where do we document `from_pretrained`'s `kwargs`?

Reply:
The best place is the docstring of `from_pretrained` above.

Reply:
Great!

Why no other `kwargs` is documented there then? There is a whole pile of those in the code.